TestClients/NICE.Identity.TestClient.NETCore/Program.cs

```diff
@@ -1,17 +1,36 @@
-﻿using Microsoft.AspNetCore;
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection.Extensions;
 
 namespace NICE.Identity.TestClient.NETCore
 {
 	public class Program
 	{
 		public static void Main(string[] args)
 		{
-			CreateWebHostBuilder(args).Build().Run();
-		}
+			Func<IHostingEnvironment, IConfigurationBuilder> configurationFactory = env =>
+				new ConfigurationBuilder()
```

Comment:
why not use CreateDefaultBuilder?

Reply:
Under the hood the same calls are being made. We have more control of the behavior by making the call this way.